DependencyInjection/Compiler/ThemeCompilerPass.php

```diff
@@ -23,6 +23,8 @@ public function process(ContainerBuilder $container)
 
         $container->setAlias('templating.cache_warmer.template_paths', 'liip_theme.templating.cache_warmer.template_paths');
 
+        $container->setAlias('twig.loader', 'liip_theme.twig.loader.filesystem');
```

Comment:
I knew that this wasn't ideal when I did it. I've had a go at doing this in a more flexible way. Do you have any thoughts on this?